### PR TITLE
Dell: S6100 fix xcvrd crash

### PR DESCRIPTION
**Pull request by aravindmani-1**
#### Why I did it
Fixes https://github.com/Azure/sonic-buildimage/issues/10038
#### How I did it
Added 10G ports in sfp_list
#### How to verify it
Check syslogs, xcvrd status.
#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [x] 202006
- [x] 202012
- [x] 202106
- [x] 202111

#### Description for the changelog
UT logs: 
[xcvrd_UT.txt](https://github.com/Azure/sonic-buildimage/files/8122046/xcvrd_UT.txt)

### CODE REVIEW

**Comment by prgeor** [comment]
@aravindmani-1 could you please run sonic-mgmt test with SFPs plugged in port 64, 65.  Attach the test result.

**Comment by aravindmani-1** [comment]
> This pull request **introduces 1 alert** when merging [446d12b](https://github.com/Azure/sonic-buildimage/commit/446d12b10be901478a7a72b1e5d6a30e214300fe) into [9fe128c](https://github.com/Azure/sonic-buildimage/commit/9fe128c8e8bb44fa9959c482116541ce64b87486) - [view on LGTM.com](https://lgtm.com/projects/g/Azure/sonic-buildimage/rev/pr-e2692332cbe53d84117f0458d3e0efc398771513)
> 
> **new alerts:**
> 
> * 1 for Wrong number of arguments in a class instantiation

This alert can be ignored as the caller function in this alert is pointing to different file.

File: platform/broadcom/sonic-platform-modules-dell/s6100/sonic_platform/chassis.py [hunk 1]
```diff
@@ -76,6 +77,14 @@ def __init__(self):
             self._module_list.append(module)
             self._sfp_list.extend(module._sfp_list)
 
+        #SFP ports
+        for index in range(64,66):
+             eeprom_path = "/sys/bus/i2c/devices/i2c-{0}/{0}-0050/eeprom".format(sfp_port)
```
Comment (tjchadaga) [inline platform/broadcom/sonic-platform-modules-dell/s6100/sonic_platform/chassis.py:82]: what is sfp_port here?

**Comment by aravindmani-1** [comment]
> @aravindmani-1 could you please run sonic-mgmt test with SFPs plugged in port 64, 65. Attach the test result.

We ran sonic-mgmt SFP test cases and all the test cases are passed.

**Comment by aravindmani-1** [comment]
> This pull request **introduces 1 alert** when merging [0202a8b](https://github.com/Azure/sonic-buildimage/commit/0202a8b683b4d45efb2c5dde918781e6b61121e2) into [9120487](https://github.com/Azure/sonic-buildimage/commit/91204879dfe8399e61ebff4c184b8df4c1da3109) - [view on LGTM.com](https://lgtm.com/projects/g/Azure/sonic-buildimage/rev/pr-24acc722d534dbee763bf3dc15d30f3071d137be)
> 
> **new alerts:**
> 
> * 1 for Wrong number of arguments in a class instantiation

This alert can be ignored as the caller function in this alert is pointing to different file.

**Review by tjchadaga — APPROVED** [review]

**Comment by qiluo-msft** [comment]
This commit could not be cleanly cherry-picked to 202012. Please submit another PR.

**Comment by aravindmani-1** [comment]
> This commit could not be cleanly cherry-picked to 202012. Please submit another PR.

Raised https://github.com/Azure/sonic-buildimage/pull/10123

**Pull request merged**